sdk.go

```diff
@@ -36,26 +36,29 @@ import (
 
 // Default ServeOptions.
 const (
-	DefaultNetwork = "tcp"
-	DefaultAddress = ":9443"
+	DefaultNetwork    = "tcp"
+	DefaultAddress    = ":9443"
+	DefaultMaxMsgSize = 4096
 )
 
 // ServeOptions configure how a Function is served.
 type ServeOptions struct {
 	Network     string
 	Address     string
+	MaxMsgSize  int
 	Credentials credentials.TransportCredentials
 }
 
 // A ServeOption configures how a Function is served.
 type ServeOption func(o *ServeOptions) error
 
-// Listen configures the network and address on which the Function will
+// Listen configures the network, address and maximum message size on which the Function will
 // listen for RunFunctionRequests.
-func Listen(network, address string) ServeOption {
+func Listen(network, address string, maxMsgSize int) ServeOption {
```

Comment:
I'd prefer to make `maxMsgSize` a new `ServeOption` rather than adding it to `Listen`.

I expect folks will need to set the max message size much less often than they'll need to set the network and address. So for most folks using `Listen`, it could be an inconvenience to have to figure out a value for this.

It's also a breaking change if we add a new required field to `Listen`, whereas adding a new `ServeOption` (e.g. `MaxMessageSize(sz int)` wouldn't be.

Reply:
I'm sorry, I didn't understand that on the first time. I added the updates right now